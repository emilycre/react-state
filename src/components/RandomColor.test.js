import React from 'react';
import { shallow } from 'enzyme';
import RandomColor from './RandomColor';

describe('RandomColor component', () => {
  it('Renders a RandomColor', () => {
    const wrapper = shallow(<RandomColor />);
    expect(wrapper).toMatchSnapshot();
  });
});
