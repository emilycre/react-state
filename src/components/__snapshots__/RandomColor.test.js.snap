// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RandomColor component Renders a RandomColor 1`] = `
<div
  style={
    Object {
      "background": "",
      "height": "25em",
      "width": "25em",
    }
  }
/>
`;
